App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Redirect, Switch } from 'react-router-dom';
+import LoginPage from './LoginPage';
+import DashboardPage from './DashboardPage';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
```

Comment:
The redirect from '/' to '/login' will always happen regardless of the user's authentication status. Consider adding a check to redirect to the dashboard if the user is already authenticated.